### DISCUSSION TITLE
OverflowError: Python int too large to convert to C long

### DISCUSSION THREAD

**Discussion by juliandwain** [asreview/asreview#1063] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hi,
thanks for this amazing library!

**Describe the bug**
I created a dataset by myself from a Scopus search and transformed it so that it can be interpreted by ASReview.
The dataset can be uploaded without any issues.
I know this is not an issue of ASReview itself, I assume that the issue is within the dataset, but I am not able to figure out why the overflow error appears.

**To Reproduce**
Upload the dataset and follow the instructions. The dataset is added as attachment.

**Screenshots**
![image](https://user-images.githubusercontent.com/38472982/166891141-286780b4-4cbe-421d-9d63-820b99485c75.png)
![image](https://user-images.githubusercontent.com/38472982/166890566-66385a8e-cb41-406d-a6cb-4ea3fd8c632f.png)

**Version information**
 - OS: Windows
 - ASReview version 0.19.1

**Additional context**
[220505_Scopus_data.csv](https://github.com/asreview/asreview/files/8629829/220505_Scopus_data.csv)

**Comment by Rensvandeschoot** [comment]
I tested your data in the beta release of version 1 and I was able to import your data without any issues. So, there is no problem with your data!!

One little detail is that all records have a label and you'll get this message:

![image](https://user-images.githubusercontent.com/36502709/166893328-5b5bc348-6822-47dd-bd4a-6e32b3005505.png)

**Comment by Rensvandeschoot** [comment, marked as answer]
see the reply by @J535D165 in https://github.com/asreview/asreview/issues/1064
> @juliandwain I can't reproduce this issue in version 0.19 on macOS.
> 
> The issue might be an integer overflow on Windows systems caused by the `record_id` column. You can rename or delete this column such that ASReview ignores it (record_ids are used by ASReview internally, but are generated on the fly if not present).
> 
> This dataset is also "fully labeled". This means that there are labels identified by ASReview. This is the result of the `label` column. If this is intentional, there is nothing to screen. If this wasn't intentional, then renaming this column is also advised.

The issue has been solved by @PeterLombaers in https://github.com/asreview/asreview/pull/1066. 

Upgrade to [v0.19.2](https://github.com/asreview/asreview/releases/tag/v0.19.2)
